edt/bms/bm_c1a0a.edt

```diff
@@ -97,26 +97,36 @@
 		{
 			"origin"	"272 28 520"
 			"angles"	"0.0 90.0 0.0"
-			"delay"		"1.5"
+			"delay"		"0.35"
 			"triggerid"	"LCS_AM_Airlock_trigger"
 			"output"	"OnTrigger"
-			"portal"	"280 -208 549"
+			"bringall"	"1"
 		}
 		"door2"
 		{
 			"origin"	"272 416 520"
-			"angles"	"0.0 90.0 0.0"
-			"delay"		"0"
+			"angles"	"7.5 80.0 0.0"
+			"delay"		"0.35"
 			"triggerid"	"door_02_btm"
-			"output"	"OnFullyClosed"
-			"portal"	"272 272 549"
+			"output"	"OnClose"
+			"bringall"	"1"
+		}
+		"hev_pickup"
+		{
+			"origin"	"925 4450 415"
+			"angles"	"12.5 -90.0 0.0"
+			"delay"		"0.0"
+			"triggerid"	"spawner_clipboard2"
```

Comment:
When I tried triggerid `HEV` and output `OnPlayerPickup` with a checkpoint, I got a SourcePawn error. Presumably spawning too late?

Reply:
Ah, it's probably a template. You gotta hook the spawner or the button which triggers it.

Reply:
If I do the button or the `point_template` that spawns in the HEV, it will be too early. The `point_template` spawns in a suit exactly once so I'm using a entity that the HEV is calling through `OnPlayerPickup`.

Reply:
Fixed by adding a new `logic_auto` into the map triggered by grabbing the HEV.